src/main/java/fi/aalto/cs/intellij/common/BuildInfo.java

```diff
@@ -0,0 +1,38 @@
+package fi.aalto.cs.intellij.common;
+
+import java.util.Properties;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Holds information that was gathered during the build of the program (such as version).
+ * Normally, this class should be accessed via {@code BuildInfo.SingletonUtility.INSTANCE}.
+ */
+public class BuildInfo {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(BuildInfo.class);
```

Comment:
Borrowing from tmc-intellij, I've used `logger` in my code. Also see [this](https://stackoverflow.com/questions/1417190/should-a-static-final-logger-be-declared-in-upper-case).

I'm fine with `LOGGER` as well, as long as we're consistent with it everywhere.